src/Workspaces/Remote/ServiceHub/Host/RemoteWorkspace.cs

```diff
@@ -254,6 +230,50 @@ async ValueTask SetLastRequestedSolutionAsync(Solution solution)
             }
         }
```

Comment:
this function is unchanged.  HOwever, it was the second of two local functions.  The first local function became an instance method and moved below.  So the diff is confused.